tests/Tests/Unit/testdata/test_service_descriptor_config.php

```diff
@@ -11,6 +11,9 @@
                     'requestPageSizeSetMethod' => 'setPageSize',
                     'responsePageTokenGetMethod' => 'getNextPageToken',
                 ],
+                'autoPopulatedFields' => [
+                    'pageToken' => \Google\Api\FieldInfo\Format::UUID4,
+                ]
```

Comment:
It's been decided that we'll use enum value instead of a string constant like `UUID4`. Ref [comment](https://github.com/googleapis/gapic-generator-php/pull/686#discussion_r1504934386)